lib/kadai-core-test/src/test/resources/fullKadai.properties

```diff
@@ -1,6 +1,7 @@
 # general configuration
 kadai.domains=DOMAIN_A | DOMAIN_B
 kadai.servicelevel.validation.enforce=false
+kadai.routing.includeOwner=true
```

Comment:
the defautl should be false, right? And shouldn't this contain the default?

Reply:
No, fullKadai.properties always contains a value different from default for testing purposes. For example, see kadai.servicelevel.validation.enforce=false. Der default für kadai.servicelevel.validation.enforce ist true